web/src/components/PageDetail.tsx

```diff
@@ -1,19 +1,27 @@
 import React, { ReactElement } from 'react'
 import styled from 'styled-components'
 
+import Link from './base/Link'
+
 const Identifier = styled.span`
   font-weight: 700;
 `
 
+const StyledLink = styled(Link)`
+  color: ${props => props.theme.colors.linkColor};
+  text-decoration: underline;
+`
+
 type PageDetailProps = {
   identifier: string
   information: string
+  pathToInformation?: string | null
```

Comment:
:upside_down_face: As the `ToInformation` does not really add any additional information, I'd prefer just `path`.
```suggestion
  path?: string | null
```